### PR TITLE
Updated `[GenTel` command to use a JSON file

### PR DESCRIPTION
**Pull request by flamewave000**
Previously all teleporters were hardcoded into the `[GenTel` command. This is not preferred. Teleporters are now contained within a CSV file that is loaded for generation.

### CODE REVIEW

**Comment by kamronbatman** [comment]
I am wondering if we should use a standard format like json. Thoughts?

**Comment by kamronbatman** [comment]
Let's use the built in one. It is supposed to be effecient.

**Comment by flamewave000** [comment]
I'll get that going, and sorry for the repeated comments. GitHub seems to be having some issues today 🤨

**Comment by flamewave000** [comment]
@kamronbatman I've got that update in. I tried to keep the JSON simple so it's not too cluttered. As such, I'm using a 3 int arrays for position data instead of an object to represent the Point3D class. So I had to write a small Point3DConverter to handle that. It adds a few lines of code, but makes for a much cleaner JSON.

Also, I know there are specialized teleporters that have things like conditions and requirements to get to certain areas. I'm not sure if those will be included in this command at some point, but the JSON route definitely makes it possible to expand if needed.

File: Projects/Scripts/Commands/GenTeleporter.cs [hunk 1]
```diff
@@ -1,1043 +1,165 @@
...
+  public class Point3DConverter : JsonConverter<Point3D>
+  {
+    public override Point3D Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
     {
-      CommandSystem.Register("TelGen", AccessLevel.Administrator, GenTeleporter_OnCommand);
+      if (reader.TokenType != JsonTokenType.StartArray)
+        throw new JsonException($"Point3D requires an array of exactly 3 integers. Invalid Starting TokenType: ${reader.TokenType}");
+      reader.Read();
+      var data = new List<int>();
+      for (; reader.TokenType != JsonTokenType.EndArray; reader.Read())
```
[the remark this answers is not in the document]
Comment (flamewave000) [reply Projects/Scripts/Commands/GenTeleporter.cs:26]: Given that I'm iterating through a data structure, does it not make sense to use the for-loop iteration paradigm?

**Pull request merged**